src/printer.js

```diff
@@ -4676,7 +4676,9 @@ function isNodeStartingWithDeclare(node, options) {
     return false;
   }
   return (
-    options.originalText.slice(0, util.locStart(node)).match(/declare\s*$/) ||
+    options.originalText
+      .slice(0, util.locStart(node))
+      .match(/declare[ \t]*$/) ||
```

Comment:
If I follow correctly, you just replaced `\s*` by `[ \t]*`. I would have thought that they were doing the same thing. Can you explain what's going on here?